detekt-core/src/main/resources/default-detekt-config.yml

```diff
@@ -700,11 +700,17 @@ style:
     active: false
   UnusedImports:
     active: false
+  UnusedParameter:
+    active: true
+    allowedNames: '(_|ignored|expected|serialVersionUID)'
```

Comment:
I don't think that `serialVersionUID` should be an allowed name for a parameter but I wanted to preserve the behavior before and after the refactor. Is this correct?